.github/workflows/feature-pr-status-check.yml

```diff
@@ -0,0 +1,36 @@
+name: ✅Feature PR Status Check
+
+
+defaults:
+  run:
+    shell: pwsh
+
+
+on:
+  pull_request:
+    branches:
+      - develop
+
+
+jobs:
+  feature_pr_status_check:
+    if: startsWith(github.head_ref, 'dependabot/nuget/') == false # Do not run with dependabot branches
```

Comment:
Remove this line.  This is residual from the dependabot stuff.  But since we are removing dependabot, it needs to be removed.

```suggestion
```